lib/stream_data.ex

```diff
@@ -214,9 +216,9 @@ defmodule StreamData do
 
       tuple_size ->
         {trees, _seed} =
-          Enum.map_reduce(0..(tuple_size - 1), seed, fn index, acc ->
+          Enum.map_reduce(1..tuple_size, seed, fn index, acc ->
             {seed1, seed2} = split_seed(acc)
-            data = elem(tuple, index)
+            data = :erlang.element(index, tuple)
```

Comment:
Does this really make a difference?

Reply:
My measurements show about 2% on big tuples (around 10 elements), so it might not be worth it.